language/de/common.php

```diff
@@ -270,6 +270,17 @@
 	'FSOCK_DISABLED'		=> 'Dieser Vorgang kann nicht abgeschlossen werden, da die <var>fsockopen</var>-Funktion deaktiviert wurde oder weil der angegebene Server nicht gefunden werden konnte.',
 	'FSOCK_TIMEOUT'			=> 'Beim Lesen des Datenstroms ist eine Zeitüberschreitung aufgetreten.',
 
+	'FILESYSTEM_CANNOT_CHANGE_FILE_GROUP'		=> 'Kann Dateigruppe nicht ändern.',
+	'FILESYSTEM_CANNOT_CHANGE_FILE_PERMISSIONS'	=> 'Kann Dateiberechtigungen nicht ändern.',
+	'FILESYSTEM_CANNOT_COPY_FILES'				=> 'Kann Dateien nicht kopieren.',
+	'FILESYSTEM_CANNOT_CREATE_SYMLINK'			=> 'Kann keinen symbolischen Link anlegen.',
+	'FILESYSTEM_CANNOT_CREATE_DIRECTORY'		=> 'Kann das Verzeichnis nicht anlegen.',
+	'FILESYSTEM_CANNOT_DELETE_FILES'			=> 'Kann Dateien nicht vom System entfernen.',
+	'FILESYSTEM_CANNOT_DUMP_FILE'				=> 'Kann Datei nicht schreiben.',
+	'FILESYSTEM_CANNOT_MIRROR_DIRECTORY'		=> 'Kann Verzeichnis nicht spiegeln.',
+	'FILESYSTEM_CANNOT_RENAME_FILE'				=> 'Kann Datei nicht umbenennen.',
+	'FILESYSTEM_CANNOT_TOUCH_FILES'				=> 'Kann nicht festellen, ob die Datei existiert.',
+
```

Comment:
Im Prinzip sind das alles die Keys mit anderem Key, die du in der acp/common.php gelöscht hast:

> 	'CANNOT_CHANGE_FILE_GROUP'	=> 'Die Gruppe einer Datei konnte nicht geändert werden',
> 	'CANNOT_CHANGE_FILE_PERMISSIONS'	=> 'Die Berechtigungen einer Datei konnten nicht geändert werden',
> 	'CANNOT_COPY_FILES'		=> 'Die Dateien konnten nicht kopiert werden',
> 	'CANNOT_CREATE_SYMLINK'	=> 'Eine symbolische Verknüpfung (Symlink) konnte nicht erstellt werden',
> 	'CANNOT_DELETE_FILES'	=> 'Eine Datei konnte nicht vom Server gelöscht werden',
> 	'CANNOT_DUMP_FILE'		=> 'Eine Datei konnte nicht ausgelesen werden',
> 	'CANNOT_MIRROR_DIRECTORY'	=> 'Ein Verzeichnis konnte nicht gespiegelt werden',
> 	'CANNOT_RENAME_FILE'	=> 'Eine Datei konnte auf dem Server nicht umbenannt werden',
> 	'CANNOT_TOUCH_FILES'	=> 'Es konnte nicht geprüft werden, ob eine Datei existiert',

Trotzdem neu übersetzen? Die neuen Versionen sind ein wenig kürzer und prägnanter, finde ich nicht schlecht.

Reply:
Die lange Version hat den Vorteil, dass sie einem deutschen Satz deutlich näher kommen als die kurze :-) Die neue ist (wie die Englische) eher abgehacktes Staccato

Reply:
Meinetwegen verwenden wir weiter die bisherige Übersetzung. Ist mir gar nicht aufgefallen. 

Allerdings sollten wir die Bedeutungsunterschiede vielleicht noch mal ansehen:
Bei DELETE_FILES ist alt Singular, bei mir Plural 
Bei Touch sind in meinen Augen beide Varianten falsch (denn touch ändert eigentlich einen Zeitstempel oder legt eine Datei an, selbst wenn die bisherige Bedeutung im Englischen korrekt ist finde ich hier aber dann meine Übersetzung besser als die bisherige). Außerdem sagt der Key Plural, die Übersetzung aber Singular.
Auch bei Dump weichen unsere Übersetzungen in der Bedeutung ab, gemeint ist meinem Verständnis nach "Daten in eine Datei schreiben".

Reply:
Ich glaube, das bei DELETE_FILES war damals Absicht. Die Meldung sollte auftreten, wenn eine Datei nicht gelöscht werden kann. Ob's dann bei weiteren geht, ist recht irrelevant. Da hänge ich aber nicht dran :-)
Beim Rest müsste man mal in den Code schauen...